captum/optim/_param/image/images.py

```diff
@@ -212,6 +259,17 @@ def forward(self) -> torch.Tensor:
 class LaplacianImage(ImageParameterization):
     """
     TODO: Fix divison by 6 in setup_input when init is not None.
+    Parameterize an image tensor with a laplacian pyramid.
```

Comment:
Do you mind adding reference doc to laplacian pyramid technique ?

Reply:
I'll add the citation in future PR as I'm having difficulty finding it at the moment. The Distill / Lucid team is also on hiatus right now, and thus I can't get answers as quickly and easily as before.